### DISCUSSION TITLE
Query Regarding Magma CWAG

### DISCUSSION THREAD

**Discussion by Aniket621** [magma/magma#15311] [Communications]
Hi all, 

Currently we are working on Magma Carrier WiFi Technology and successfully build CWAG, FeG, Mock HSS, FreePCRF and all container and Link b/w CWAG-FeG and FeG-HSS,FeG-PCRF are up. Has anyone tried to configure Cisco Catalyst AP 9105AX or any other AP vendor  Model integrated with CWAG? Also, has anyone tried configuring the EoGRE tunnel b/w AP and Gateway.

**Comment by panyogesh** [comment]
@Aniket621 : As part of regular magma activity CWAG is just integration tested. Not sure if anybody is specifically testing it with the hardware mentioned.  If you are facing any specific issue from protocol perspective kindly raise an issue.

**Comment by panyogesh** [comment]
Lets for a week to see if there is anyone in the community who has tried the same.

**Comment by lucasgonze** [comment]
@Aniket621 Were you able to test?

**Comment by panyogesh** [comment]
Closing the discussion.
